### PR TITLE
Lint Codebase

### PR DESCRIPTION
**Pull request by sumitra19jha**
**What kind of change does this PR introduce?**
bugfix and refactoring. Issue: #691 and #400 

**Did you add tests for your changes?**
Separate test are not required here

**If relevant, did you update the documentation?**
Yes

**Summary**
1. To fix all dart analyzer issues.
2. Addition of linter
3. To make sure all files follow the instruction specified in linter
4. `flutter format .` command in the codebase

**Does this PR introduce a breaking change?**
No

**Other information**
Demo of app after changes

https://user-images.githubusercontent.com/35868598/115825923-bdcde800-a427-11eb-8fd1-18cf4470f872.mp4

### CODE REVIEW

**Comment by Sagar2366** [comment]
@sumitra19jha excellent work.
Can you fix lint issues coming in pubspec.yaml file as well ?

**Comment by sumitra19jha** [comment]
Yeah working on this

**Comment by sumitra19jha** [comment]
Please review the PR. The issue is fixed

File: lib/views/pages/newsfeed/addPost.dart [hunk 1]
```diff
@@ -1,210 +0,0 @@
-//flutter imported packages
```
Comment (Sagar2366) [inline lib/views/pages/newsfeed/addPost.dart:1]: Instead of deleting entire file, can you add only changes required to fix the linter issue ?

**Review by Sagar2366 — APPROVED** [review]

**Comment by Sagar2366** [comment]
Kudos to @sumitra19jha for work on this Issue and PR.

**Comment by sumitra19jha** [comment]
Thank you😊

**Pull request merged**